net/GCDWebServer/Portfile

```
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           xcode 1.0
+PortGroup           github 1.0
+
+github.setup        swisspol GCDWebServer 3.5.4
+categories          net
+platforms           darwin
+license             BSD
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         ${name} is a modern and lightweight HTTP 1.1 server based on Grand Central Dispatch.
+long_description    ${name} is a modern and lightweight HTTP 1.1 server based on Grand Central Dispatch. \
+                    It was written from scratch with the following goals in mind: \
+                    \n \
+                    \n* Elegant and easy to use architecture with only 4 core classes: server, \
+                    connection, request and response \
+                    \n* Well designed API with fully documented headers for easy integration and customization \
+                    \n* Entirely built with an event-driven design using Grand Central Dispatch \
+                    for best performance and concurrency \
+                    \n* No dependencies on third-party source code \
+                    \n* Available under a friendly New BSD License
+
+checksums           rmd160  daf4ccdd0ee8f6aa160597c73bd6f1f3903fd31b \
+                    sha256  a2187e56572aad181e2ee642805cbc3f94f13847e857a65c9204c7f5b4f4531b \
+                    size    10814649
+
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} @${version} requires macOS 10.7 or greater"
+        return -code error "incompatible OS X version"
```

10.7 and earlier were called "Mac OS X" not "OS X" nor "macOS".
```suggestion
        ui_error "${name} @${version} requires Mac OS X 10.7 or greater"
        return -code error "incompatible Mac OS X version"
```

---

Done